script/Arcueid/Day 7/07_00_ARC07_3D.txt

```diff
@@ -91,7 +91,7 @@ That wasn't me, it was the work of a different vampire."
 [sha:b63ad4b6d1edf740f1c6fb36212a66b797b80b1e]{
 -- Page 234, Offset 17142.
 -- あっさりと緊張を解いて、呆れたようにアルクェイドは即答した。
-Appalled, Arcueid answers immediately, dispelling the surrounding tension with ease.
+Appalled, she answers immediately, dispelling the surrounding tension with ease.
```

Comment:
We may need to discuss this for all the VN, Shiki almost always mentions Arcueid by name, let alone in his head, it's starting to look off for me.